<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALCULATOR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../../../Pictures/Projet_image_Ministry/minfi_trademark_without_shadow.png</normalon>
   </iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>120</width>
    <height>120</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(206, 227, 255);
text-align:right;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: #00ff00;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_substract">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Multi">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_16:pressed{
   background-color:  qlineargradient(x1:0,y1:0,x2:0,y2:1,
                                   stop:0 #dadbde, stop : 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PlusMinus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
